lusidtools/cocoon/utilities.py

```diff
@@ -323,15 +259,22 @@ def set_attributes_recursive(
     """
     if total_count == none_count or missing_value:
         return None
+    for key, value in obj_init_values.items():
```

Comment:
change to allow constant values, for strings check for starting with `$` for constant value